docs/troubleshooting/clusterResourcePlacementAvailable.md

```diff
@@ -0,0 +1,205 @@
+# How can I debug when my CRP ClusterResourcePlacementAvailable condition is set to false?
+The ClusterResourcePlacementAvailable condition is false when the cluster lacks the necessary resources or capabilities to accommodate new deployments or allocations.

```

Comment:
"lacks the necessary resources or capabilities to accommodate new deployments or allocations" the resource cannot be available for other reasons as well Ex. like bad image name in the spec for deployment